src/styles/variables.css

```
@@ -0,0 +1,18 @@
+:root {
+  --main-font-color: #060b35;
+  --secondary-font-color: #616070;
+
+  --button-color: #00acdc;
+
+  --font-size-small: 10px;
+  --font-size-medium: 12px;
+
+  --font-height-medium: 14px;
+  --font-height-large: 18px;
```

For consistancy I would name those variable like this:
```suggestion
  --font-size-small: 10px;
  --font-size-medium: 12px;

  --font-height-small: 14px;
  --font-height-medium: 18px;
```